src/storage/hummock_sdk/src/sstable_info.rs

```diff
@@ -67,6 +87,53 @@ impl SstableInfoInner {
     }
 }
 
+impl From<&PbVnodeStatistics> for VnodeStatistics {
+    fn from(info: &PbVnodeStatistics) -> Self {
+        Self {
+            vnode_key_ranges: info
+                .vnode_key_ranges
+                .iter()
+                .map(|range| {
+                    let min_key = FullKey::decode(&range.min_key).copy_into();
+                    let max_key = FullKey::decode(&range.max_key).copy_into();
```

Comment:
Better add an assertion either here or in meta or both to make sure
1. min_key and max_key share the same table id
2. min_key and max_key share the same vnode